apps/client/src/components/Questions.js

```diff
@@ -114,68 +116,67 @@ const Questions = ({ checker, topic: { slug } }) => {
     checker.next();
   }
 
-  // @TODO: Refactor this map function
-  return checker.stack.filter(uniqueFilter).map((q, i) => {
-    // Define userAnswer
-    let userAnswer;
-    if (q.options) {
-      userAnswer = q.answer;
-    } else {
-      const responseObj = booleanOptions.find((o) => o.value === q.answer);
-      userAnswer = responseObj?.label;
-    }
+  return (
+    checker.stack
+      // Remove duplicate questions (do we really need this?)
+      .filter(uniqueFilter)
```

Comment:
We don't need this anymore, autofill is fixed.

Reply:
👍